main.nf

```diff
@@ -1,15 +1,16 @@
 // Ensure DSL2
 nextflow.enable.dsl = 2
 
-//url for exmaple CWL workflow
+//url for example CWL workflow
 params.cwl_file = "https://raw.githubusercontent.com/CRI-iAtlas/iatlas-workflows/develop/EPIC/workflow/steps/epic/epic.cwl"
```

Comment:
I was planning on doing that in a future version, but at the moment these params are the only way it will run successfully without changing any of the existing cwl workflows